src/monitors.js

```diff
@@ -29,9 +37,13 @@ module.exports.spotRequestSubmitted = (monitor, series, provisionerId,
     price: bid.truePrice,  // ugh, naming!
     bias: bid.bias,
   });
-  monitor.measure(`${region}.${az}.${instanceType}.${workerType}.spot.bid`, bid.price);
-  monitor.measure(`${region}.${az}.${instanceType}.${workerType}.spot.price`, bid.truePrice);
-  monitor.measure(`${region}.${az}.${instanceType}.${workerType}.spot.bias`, bid.bias);
+  monitor.measure('overall.price-per-capacity-unit', bid.truePrice);
+  monitor.measure(`${region}.${az}.${instanceType}.overall.spot.bid`, bid.price);
+  monitor.measure(`${region}.${az}.${instanceType}.overall.spot.price`, bid.truePrice);
+  monitor.measure(`${region}.${az}.${instanceType}.overall.spot.bias`, bid.bias);
+  monitor.measure(`${region}.${az}.${instanceType}.worker.${workerType}.spot.bid`, bid.price);
+  monitor.measure(`${region}.${az}.${instanceType}.worker.${workerType}.spot.price`, bid.truePrice);
+  monitor.measure(`${region}.${az}.${instanceType}.worker.${workerType}.spot.bias`, bid.bias);
```

Comment:
Consider not adding the region, or submitting both with and without region/az

Often times it's easier to look at fewer dimensions... And then explore deeper...